src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -187,6 +187,23 @@ public boolean isDependentArgumentAllowed(final Class<?> dependentClass) {
         return filters;
     }
 
+    /**
+     * Get the list of default plugins used for this instance of this descriptor. Used for help/doc generation.
+     * @return List of T
+     */
+    @Override
+    public List<Object> getDefaultInstances() { return new ArrayList<>(toolDefaultReadFilters.values()); }
```

Comment:
@magicDGS I see the distinction you're making, but I think thats a different method. The intent of this method is that it returns the "list of defaults used by the tool", not the "list of defaults that are left after arguments are parsed". It should return the same list before and after parsing.

I know you're anxious to get your changes into the code base, but I think the next step will be for me to create a Barclay PR that gets the interface shaped approximately the way we want it based on all of the discussion we've had (we already have two proposed name changes to implement). I hope that helps.